src/HotChocolate/Data/test/Data.Filters.InMemory.Tests/FilterVisitorTestBase.cs

```diff
@@ -28,26 +28,25 @@ public FilterVisitorTestBase()
             TEntity[] entities,
             FilterConvention? convention = null)
             where TEntity : class
-            where T : IFilterInputType
+            where T : FilterInputType<TEntity>
         {
-            convention ??= new FilterConvention(x => x.UseDefault());
+            convention ??= new FilterConvention(x => x.UseDefault().BindRuntimeType<TEntity, T>());
```

Comment:
@michaelstaib  this is currently a limitation. The types have to be registered on the convention

Reply:
Lets look at this in a follow up.